### ISSUE TITLE
Update logo: Paramount Channel

### ISSUE THREAD

**Issue by manuelrn** [picons/picons#61]
Hi,
'Paramount Channel' has changed the name to 'Paramount Network', and therefore also its logo.

`srp.index`
75F9_408_1_C00000=paramountchannel

The new logo:
https://github.com/picons/picons/blob/master/build-source/logos/paramountnetwork.default.svg

In the repository, there is a single file from the old channel ([paramountchannel.default.svg](https://github.com/picons/picons/blob/master/build-source/logos/paramountchannel.default.svg)), because it is blue and that color is valid for both dark and light.

But for the new channel there are two files ([paramountnetwork.default.svg](https://github.com/picons/picons/blob/master/build-source/logos/paramountnetwork.default.svg) and [paramountnetwork.light.svg](https://github.com/picons/picons/blob/master/build-source/logos/paramountnetwork.light.svg)), when really the 'default' svg is also blue just like old Paramount Channel, and therefore it is valid for both dark and light so the 'light' file should not exist.

Could you delete the light svg for this reason? So, in any way, it is generated in blue color.

Thank you very much ;)

**Comment by mrhotio** [comment]
Please, just give us the correct additions to srp and snp index...there's probably a good reason for the light version.

**Comment by manuelrn** [comment]
> Please, just give us the correct additions to srp and snp index...there's probably a good reason for the light version.

It would be necessary to change this actual channel:
```75F9_408_1_C00000=paramountchannel```

Thanks!

**Comment by AbuBaniaz** [comment]
A change involves altering something from A to B. Or deleting one thing and repalcing it with another.

I have changed from (deleted) 
```
paramount=paramountchannel`
75F9_408_1_C00000=paramountchannel
```
to  (added)
```
paramount=paramountnetwork
75F9_408_1_C00000=paramountnetwork
```

**Comment by Lexzie** [comment]
Reading this I had to look into it.
The mentioned service reference is that of Paramount Network España.
Definitely not the only version of Paramount Channel/Network.

**Still Paramount Channel:**
Paramount Channel France (at least on their [website](https://www.paramountchannel.fr/))
Paramount Channel Hungary
Paramount Channel Polska
Paramount Channel Romania
Paramount Channel Russia (at least on their [website](https://paramountchannel.ru/))

**Changed to Paramount Network**
Paramount Network Italia
Paramount Network UK

So I changed a few more links.
